pyatlan/model/packages/asset_export(basic).py

```diff
@@ -0,0 +1,290 @@
+        """
+        self._export_scope = "GLOSSARIES_ONLY"
+        self._parameters.append({"name": "export_scope", "value": self._export_scope})
+        self._parameters.append(
+            {
+                "name": "include_archived",
+                "value": include_archived,
+            }
+        )
+        return self
+
+    def enriched_only(
+        self,
+        prefix: str,
+        include_description: Optional[str] = None,
+        include_glossaries: Optional[bool] = None,
+        include_data_products: Optional[bool] = None,
+        include_archived: Optional[bool] = None,
+    ) -> AssetExportPackage:
+        """
+        Set up the package to export only enriched assets.
+        :param prefix: Qualified name prefix to filter assets.
+        :param include_description: Whether to include asset descriptions in the export.
+        :param include_glossaries: Whether to include glossaries in the export.
+        :param include_data_products: Whether to include data products in the export.
+        :param include_archived: Whether to include archived assets in the export.
+        """
+        self._export_scope = "ENRICHED_ONLY"
+        self._parameters.append({"name": "export_scope", "value": self._export_scope})
+        params = {
+            "qn_prefix": prefix,
+            "include_description": include_description,
+            "include_glossaries": include_glossaries,
+            "include_data_products": include_data_products,
+            "include_archived": include_archived,
+        }
+        self._add_optional_params(params)
+        return self
+
+    def products_only(
+        self, include_archived: Optional[bool] = None
+    ) -> AssetExportPackage:
+        """
+        Set up the package to export only data products.
+        :param include_archived: Whether to include archived data products in the export.
+        """
+        self._export_scope = "PRODUCTS_ONLY"
+        self._parameters.append({"name": "export_scope", "value": self._export_scope})
+        self._parameters.append(
+            {
+                "name": "include_archived",
+                "value": include_archived,
+            }
+        )
+        return self
+
+    def all_assets(
+        self,
+        prefix: str,
+        include_description: Optional[str] = None,
+        include_glossaries: Optional[bool] = None,
+        include_data_products: Optional[bool] = None,
+        include_archived: Optional[bool] = None,
+    ) -> AssetExportPackage:
+        """
+        Set up the package to export all assets.
+        :param prefix: Qualified name prefix to filter assets.
+        :param include_description: Whether to include asset descriptions in the export.
+        :param include_glossaries: Whether to include glossaries in the export.
+        :param include_data_products: Whether to include data products in the export.
+        :param include_archived: Whether to include archived assets in the export.
+        """
```

Comment:
Same for every method docstrings: https://github.com/atlanhq/atlan-python/pull/402/files#r1818759623